packages/api/tests/task.test.js

```diff
@@ -2047,11 +1996,11 @@ describe('Task tests', () => {
 
   describe('Patch task date test', () => {
     test('Owner must be able to patch task date', async (done) => {
-      const date = faker.date.future().toISOString().split('T')[0];
+      const due_date = faker.date.future().toISOString().split('T')[0];
```

Comment:
@MwayaB I made a few fixes related to unknown variables. Here, I changed `date` to `due_date` because the (now commented out) console.log and other statements errored with `due_date` not defined.